docs/synapse/kubernetes/aha.yaml

```diff
@@ -57,26 +57,26 @@ spec:
       volumes:
         - name: data
           persistentVolumeClaim:
-            claimName: example-aha
+            claimName: example-aha00
       containers:
-      - name: aha
+      - name: aha00
         image: vertexproject/synapse-aha:v2.x.x
         env:
           - name: SYN_LOG_LEVEL
             value: DEBUG
           - name: SYN_LOG_STRUCT
             value: "false"
-          - name: SYN_AHA_AHA_NAME
-            value: aha
+          - name: SYN_AHA_DNS_NAME
+            value: aha00.default.svc.cluster.local
           - name: SYN_AHA_AHA_NETWORK
             # This is <namespace>.<cluster dns root> - it is used as Certificate Authority name
-            value: default.svc.cluster.local
+            value: dev.synapse
           - name: SYN_AHA_DMON_LISTEN
             # This is <aha name>.<namespace>.<cluster dns root> and the CA name from above
-            value: "ssl://0.0.0.0?hostname=aha.default.svc.cluster.local&ca=default.svc.cluster.local"
```

Comment:
We can also remove the `dmon:listen` and `provision:listen` examples here since we're now listening on 0.0.0.0 and setting the hostname values to the `dns:name` values. See https://github.com/vertexproject/synapse/blob/v2.202.0/synapse/lib/aha.py#L851-L873